Pushbullet/Configuration/PluginConfiguration.cs

```diff
@@ -18,6 +18,6 @@ public class PluginConfiguration : BasePluginConfiguration
         /// Gets or sets the configured options.
         /// </summary>
         /// <returns><see cref="IEnumerable{PushbulletOptions}"/>.</returns>
-        public PushbulletOptions[] Options { get; set; } = Array.Empty<PushbulletOptions>();
+        public IEnumerable<PushbulletOptions> Options { get; set; } = Array.Empty<PushbulletOptions>();
```

Comment:
I'm pretty sure this change isn't supported by the xml serializer... best is to just disable the warning in this file

Reply:
NGL, I personally would prefere if the configs where in something more readable than XML but then thats not a discussion for here nor any time soon :stuck_out_tongue_closed_eyes: